stable/proxysql/templates/deployment.yaml

```diff
@@ -51,6 +51,9 @@ spec:
               containerPort: 6070
               protocol: TCP
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          - name: USE_AWS_AURORA
```

Comment:
isn't it better to copy all envs from values?

Reply:
I wanted to to keep that specific variable separate from other environment variable

Reply:
I can add separate env variable block that can be templated. But for sure I want to keep this variable separate from that